sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -554,6 +555,8 @@ def _convert_to_tuple(cls, table_field_schemas):
             for x in table_field_schemas]
 
   def decode(self, value):
+    if isinstance(value, bytes):
```

Comment:
Is the value here always `bytes`? If so, do we need the if? In general it is strongly discouraged to have apis that can take both bytes and strings.

Reply:
Unless @kamilwu is available to resolve this soon, I'm inclined to just file a folow-up jira and merge this as is to get postcommits green.

Reply:
Indeed, it's unnecessary. Pushing a fix